GLAA.Services/PublicRegister/IPublicRegisterViewModelBuilder.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using GLAA.ViewModels.PublicRegister;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace GLAA.Services.PublicRegister
+{
+    public interface IPublicRegisterViewModelBuilder
+    {
+        List<SelectListItem> UkCountries { get; }
+        PublicRegisterLicenceListViewModel BuildAllLicences();
+        PublicRegisterLicenceListViewModel BuildEmptySearch();
+        PublicRegisterLicenceSummaryViewModel BuildLicence(int id);
+        PublicRegisterLicenceListViewModel BuildSearchForLicences(PublicRegisterSearchViewModel publicRegisterSearchViewModel);
```

Comment:
What's this guy building? wouldn't the VM param here be passed to a post data handler?